test/integration/volume/attach_detach_test.go

```diff
@@ -128,6 +129,7 @@ func TestPodDeletionWithDswp(t *testing.T) {
 		t.Fatalf("Pod not found in Pod Informer cache : %v", err)
 	}
 
+	waitForPodsInDSWP(t, ctrl.GetDesiredStateOfWorld())
 	podsToAdd := ctrl.GetDesiredStateOfWorld().GetPodToAdd()
 
 	if len(podsToAdd) == 0 {
```

Comment:
These 3 lines are now redundant.  ``waitForPodsInDSWP`` will fail if no pods can be found by the timeout.